compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaRecordTypeSymbol.java

```diff
@@ -103,8 +103,21 @@ public BallerinaRecordTypeSymbol(CompilerContext context, ModuleID moduleID, BRe
 
     @Override
     public String signature() {
+        // if isInclusive and has a rest type desc, then the rest type desc must be ANY_DATA.
+        // otherwise it is actually exclusive.
+        // This is a counter for following being identified as inclusive.
+        // type Person record {|
+        //    string name;
+        //    int age;
+        //    string...;
+        //|};
+        boolean isRestAnyData = restTypeDescriptor()
+                .map(t -> TypeDescKind.ANYDATA.equals(t.typeKind()))
+                .orElse(true);
```

Comment:
Missing rest type desc does not imply that the rest type is `anydata`. A closed record doesn't have a rest type desc.

Reply:
Yes, but this is actually representing `is rest anydata if there is a rest type descriptor`. So if there isn't a rest type descriptor, the value should be true. Maybe I have to rename it?